docs/source/quick-start.ipynb

```diff
@@ -1009,16 +1009,16 @@
        "    observations_category_bounds  (observations_category) &lt;U10 &#x27;[0.0, 0.5)&#x27; &#x27;...\n",
```

Comment:
<p>FYI, when I execute this cell, I get <code>nan</code>  for <code>mae_hwci</code>  unless I make <code>observations=None</code>.  This might be on my end. I'm having trouble checking out your environment properly so I had to paste your edits into my environment manually.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xskillscore/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
I dont get this/ also `observations_category_bounds` seems somewhere else